### PR TITLE
More CR comments from PR#85

### PR DESCRIPTION
**Pull request by pgilmorepf**

### CODE REVIEW

**Review by raymondchams — APPROVED** [review]

File: LocalMultiplayerAgent/SettingsJsonGenerator/scripts.js [hunk 1]
```diff
@@ -2,6 +2,27 @@
 let BuildGuId = crypto.randomUUID();
 let SessionGuId = crypto.randomUUID()
 
+// These values match an enum for RunMode in the HTML file
+let RUN_MODE_WIN_PROCESS = "WinProcess";
```
Comment (nassosterz-ms) [inline LocalMultiplayerAgent/SettingsJsonGenerator/scripts.js:6]: Maybe have all these be const?

Comment (raymondchams) [reply LocalMultiplayerAgent/SettingsJsonGenerator/scripts.js:6]: I think 'let' is const in js.

**Review by dgkanatsios — APPROVED** [review]

**Comment by emmayspark** [comment]
Are we going to remove the current Json configuration permanently or supporting both?

File: LocalMultiplayerAgent/SettingsJsonGenerator/scripts.js (hunk 1 above)
Comment (pgilmorepf) [reply LocalMultiplayerAgent/SettingsJsonGenerator/scripts.js:6]: No. It's only about scoping.
https://stackoverflow.com/questions/762011/whats-the-difference-between-using-let-and-var
Basically let is more precisely similar to the way ALL C++/C#/etc languages work, where var is abusing the underlying language implementation in ways that "shouldn't technically work, but do".

**Comment by pgilmorepf** [comment]
E: This doesn't replace it. It's a generator for it. The Json configuration stays, but this webpage can generate it from a simple gui of options, which also has range-checking, validation, and other helpful inputs to ensure you have input the correct values.

File: LocalMultiplayerAgent/SettingsJsonGenerator/scripts.js (hunk 1 above)
Comment (nassosterz-ms) [reply LocalMultiplayerAgent/SettingsJsonGenerator/scripts.js:6]: I am not familiar with JS, but what about this? https://www.tutorialspoint.com/const-vs-let-in-javascript

Comment (pgilmorepf) [reply LocalMultiplayerAgent/SettingsJsonGenerator/scripts.js]: I was wrong! Fixed.

**Pull request merged**